frontend/javascripts/libs/react_hooks.ts

```diff
@@ -90,6 +90,39 @@ export function useKeyPress(targetKey: "Shift" | "Alt" | "Control") {
   return keyPressed;
 }
 
+export function useRepeatedButtonTrigger(triggerCallback: () => void, repeatDelay: number = 150) {
+  const [isPressed, setIsPressed] = useState(false);
+
+  useEffect(() => {
+    let timerId: NodeJS.Timeout;
```

Comment:
Minor, but people seem to be using [this](https://stackoverflow.com/a/51040768) type instead.